demo/deploy-zookeeper.sh

```diff
@@ -24,7 +24,31 @@ if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
     fi
 fi
 
-kubectl apply -nzookeeper -f - <<EOF
+kubectl apply -f - <<EOF
+apiVersion: specs.smi-spec.io/v1alpha4
+kind: TCPRoute
+metadata:
+  name: zookeeper
+  namespace: zookeeper
+spec:
+  matches:
+    ports:
+    - 2181
+    - 3181
+---
+apiVersion: specs.smi-spec.io/v1alpha4
+kind: TCPRoute
+metadata:
+  name: zookeeper-internal
+  namespace: zookeeper
+spec:
+  matches:
+    ports:
+    - 2181
+    - 3181
+    - 2888
+    - 3888
+---
 kind: TrafficTarget
```

Comment:
Just a thought, should the deploy-zookeeper script call deploy-kafka? The zookeeper TrafficTarget has no effect until kalfka has been deployed (which is totally fine)

Reply:
Well, the TrafficTarget allows the zookeepers to talk to each other as well, so it is necessary in that first zookeeper deployment step. I also think it helps to mentally follow what's going on